src/BlazorUI/Bit.BlazorUI/Components/Inputs/DatePicker/BitDatePicker.scss

```
@@ -1,4 +1,4 @@
-﻿@import "../../../Styles/functions.scss";
+@import "../../../Styles/functions.scss";
```

Stylelint (17.4.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)